src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -11,6 +11,8 @@ namespace System.Net.Tests
 {
     public class HttpListenerPrefixCollectionTests
     {
+        public static bool IsNonZeroLowerBoundArraySupported { get; } = PlatformDetection.IsNonZeroLowerBoundArraySupported;
```

Comment:
I changed to `{ get; }` in order to follow the pattern in the original file.

We can open a new issue if `=>` is the preferred behavior.